packages/ui/src/components/va-alert/VaAlert.vue

```diff
@@ -80,6 +80,7 @@ export default defineComponent({
     ...useStatefulProps,
     modelValue: { type: Boolean as PropType<boolean>, default: true },
     color: { type: String as PropType<string>, default: 'primary' },
+    textColor: { type: String as PropType<string>, default: '' },
```

Comment:
No need to wrap primitives with PropType<> (type: String as PropType<string> === type: String)

```suggestion
    textColor: { type: String, default: '' },
```

We're going to clear away such cases soon :)

Reply:
> No need to wrap primitives with PropType<> (type: String as PropType === type: String)
> 
> We're going to clear away such cases soon :)

I would suggest make this kind of cleanup in different task.